flow-base-api/src/main/java/com/aevi/sdk/flow/BaseApiClient.java

```diff
@@ -158,6 +179,14 @@ protected ChannelClient getMessengerClient(ComponentName componentName) {
         }
     }
 
+    protected Throwable createFlowException(Throwable throwable) {
+        if (throwable instanceof MessageException) {
+            MessageException e = (MessageException) throwable;
+            return new FlowException(e.getCode(), e.getMessage());
+        }
+        return throwable;
```

Comment:
err sorry - late comment. So should we not map any non-FlowException ones here? Like `new FlowException(UNKNOWN_ERROR, throwable.getMessage)` but maybe a new constructor and pass in throwable as the "cause"?